lib/src/api/api.dart

```diff
@@ -1,7 +1,10 @@
 import 'package:http/http.dart' as http;
+import 'package:retry/retry.dart';
 
 import 'gateway_common.dart' if (dart.library.html) 'gateway_web.dart';
 
+const MAX_RETRIES = 10;
```

Comment:
What's the max backoff time based on this number? It's useful to know since it'll tell you how long it might take for the web app to give up on an upload. FWIW longer is probably better since we don't do local tx caching right now (i.e. no upload resumption after disruptions). But there could definitely be an amount of wait time that is "too long".

Reply:
400 ms for the first retry and it doubles after every attempt, 400ms, 800ms, 1600ms, 3200ms and so on for 10 retries.